F21/cemilcan98/app.py

```diff
@@ -0,0 +1,84 @@
+def toLowerCase(i):
+    if(i[1] in ["en", "en-US", "en-IE", "en-Latn"]):

```

Comment:
The code should be much more general than just the examples in the test file. Worth reading through the BCP-47 standard for this.